test/browser/main.fmf

```diff
@@ -14,7 +14,8 @@ require:
   - python3-yaml
   # required by tests
   - firewalld
-  - libvirt-daemon-config-network
+  - libvirt-daemon-driver-storage-iscsi
+  - libvirt-daemon-driver-storage-logical
```

Comment:
I understand other changes where we replace general package with more specific ones, but what is the motivation for this change?
`libvirt-daemon-config-network` and `libvirt-daemon-driver-storage-*` seem to serve different purposes. 
Is libvirt-daemon-config-network redundant? are libvirt-daemon-driver-storage-* suddenly needed?

Reply:
The removal and addition are independent from each other, they just happened to be at the same place. As I wrote in the commit message, libvirt-daemon-config-network got moved to the .spec:

    Requires: (libvirt-daemon-config-network if virt-install)

as I noticed that creating a system VM fails on the missing default network. So it really shouldn't be a test-only dependency. I think I'll move this into a separate commit to explain it better.

The tests cover iscsi and LVM storage pools. They were previously pulled in via libvirt-daemon-kvm, but I replaced the storage parts with libvirt-daemon-driver-storage-core. That will just give you basic directories, but avoids pulling in iSCSI or LVM (which include udev rules and running services!) on machines which don't need/want it.

Reply:
I split out the config-network commit, hopefully that clarifies it a bit?

Reply:
Ah yeah thanks, I just thought the removal of the network package and the addition of the storage package are related to each other, and that confused me